metasynth/__main__.py

```diff
@@ -0,0 +1,88 @@
+"""CLI for generating synthetic data frames from a metasynth .json file."""
+import argparse
+import pathlib
+import pickle
+import sys
+
+from metasynth._version import __version__
+from metasynth import MetaFrame
+
+
+def main():
+    """Parse arguments and generate synthetic data."""
+    parser = argparse.ArgumentParser(
+        prog="metasynth",
+        description="Synthesize data from Generative Metadata Format .json file.",
+    )
+    parser.add_argument(
+        "input",
+        help="input file; typically .json adhering to the Generative Metadata Format",
+        type=pathlib.Path
+    )
+    parser.add_argument(
+        "output",
+        help="output file (.csv, .feather, .parquet, .pkl, or .xlsx)",
+        nargs="?",
+        type=pathlib.Path
+    )
+    parser.add_argument(
+        "-n", "--num_rows",
+        help="number of rows to synthesize",
+        default=None,
+        type=int,
+        required=False
+    )
+    parser.add_argument(
+        "-p", "--print_only",
+        help="print six-row data frame to console and exit",
```

Comment:
```suggestion
        help="preview six-row synthesized data frame to console and exit",
```

Maybe changing it to `--preview` makes sense?